pkg/apis/cartographer/v1alpha1/workload_helpers.go

```diff
@@ -425,6 +425,15 @@ func (w *Workload) MergeAnnotations(key, value string) {
 	w.Annotations[key] = value
 }
 
+func (w *Workload) RemoveAnnotations(key string) {
+	delete(w.Annotations, key)
+}
+
+func (w *Workload) AnnotationExists(key string) bool {
```

Comment:
*nit:* Would you like to call this `IsAnnotationExists` ?

Reply:
ok, changed